experimental/auth-logic/verifier_wrapper_test.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package authlogic contains logic and tests for interfacing with the
+// authorization logic compiler
+package authlogic
+
+import (
+	"testing"
+)
+
+func TestVerifierWrapper(t *testing.T) {
+	want :=
+		`"OakFunctionsLoader::Verifier" says {
+"OakFunctionsLoader::EndorsementFile" canSay expected_hash("OakFunctionsLoader::ProvenanceFile", any_hash).
```

Comment:
It looks like the endorsement files no longer list a hash for the provenance file. I'll change this shortly so that this statement is no longer here and instead there is another `ProvenanceFile` principal that also gives an expected hash of the binary and I'll change the rule at the bottom to check that the provenance file and the endorsement file give the same hash. (The `ProvenanceFile` principal will be distinct from the `ProvenanceFileBuilder` -- the builder one is the result of trying to build the using the provenance file and the `ProvenanceFile` one just parses a hash from it).

Reply:
fixed. @rbehjati ready for review now